ext/strings.go

```diff
@@ -433,6 +434,11 @@ func (sl *stringLib) CompileOptions() []cel.EnvOption {
 					d := delim.(types.String)
 					return stringOrError(joinSeparator(l.([]string), string(d)))
 				}))),
+		cel.Function("strings.quote", cel.Overload("strings_quote", []*cel.Type{cel.StringType}, cel.StringType,
```

Comment:
Also, don't forget to update the doc comment and associated README.md with the same usage information. Thank you!

Reply:
Gotcha, godoc at the top of the file and README have both been amended.